boards/arm/mec15xxevb_assy6853/Kconfig.defconfig

```diff
@@ -72,4 +72,32 @@ config I2C_XEC_2
 
 endif # I2C
 
+if SYS_CLOCK_EXISTS
+# If the kernel has timer support, enable both clock control and timer
+config CLOCK_CONTROL
+	default y
+
+config MCHP_XEC_RTOS_TIMER
+	default y
+
+endif # SYS_CLOCK_EXISTS
+
+config SYS_POWER_MANAGEMENT
```

Comment:
Remove the Kconfig flags and code changes related to power management, and create a separate PR to investigate power management.

Reply:
Done